src/LessonPreview/LessonPreview.js

```diff
@@ -13,19 +13,40 @@ const LessonPreview = ({
   deleteQuestion,
   addLesson,
 }) => {
-
+  const teacherId = 1
   const isEnabled = lessonTitleText.trim() === '' || questions.length < 1
 
   const createLesson = (e) => {
     e.preventDefault();
+    const formattedQuestions = questions.map(question => {
+      return {
+        question: question.question,
+        reading: question.reading,
+        answers: question.answers
+      }
+    })
     const lesson = {
-      name: lessonTitleText,
-      questions: questions,
+      lesson: {
+        name: lessonTitleText,
+        questions: formattedQuestions,
+      }
```

Comment:
Interesting - this got updated with the backend?

Reply:
Yes this was the issue all along.